### PR TITLE
prepend table name for pagination ordering #980

### PR DESCRIPTION
**Pull request by chapmandu**

### CODE REVIEW

File: wheels/model/read.cfm [hunk 1]
```diff
@@ -79,7 +79,7 @@ public any function findAll(
 			for (local.i = 1; local.i <= local.iEnd; local.i++) {
 				local.item = primaryKeys(local.i);
 				if (!ListFindNoCase(local.compareList, local.item) && !ListFindNoCase(local.compareList, tableName() & "." & local.item)) {
-					arguments.order = ListAppend(arguments.order, local.item);
+					arguments.order = ListAppend(arguments.order, "#tableName()#.#loc.item#");
```
Comment (chapmandu) [inline wheels/model/read.cfm]:
```suggestion
					arguments.order = ListAppend(arguments.order, "#tableName()#.#local.item#");
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,6 +42,7 @@
 - Fixes bug where Migrator \$execute() always appends semi-colon - [#924](https://github.com/cfwheels/cfwheels/issues/#924) [Adam Chapman]
 - Fixes bug where model createdAt property is changed upon update - [#927](https://github.com/cfwheels/cfwheels/issues/#927) [Brandon Shea, Adam Chapman]
 - Fixed silent application.wheels scope exception hampering autoMigrateDatabase - [#957](https://github.com/cfwheels/cfwheels/issues/#957) [Adam Chapman, Tom King]
+- Fixed pagination order ambiguous column name exception - [#957](https://github.com/cfwheels/cfwheels/issues/#980) [Adam Chapman, Mike Lange]
```
Comment (chapmandu) [inline CHANGELOG.md]:
```suggestion
- Fixed pagination order ambiguous column name exception - [#980](https://github.com/cfwheels/cfwheels/issues/#980) [Adam Chapman, Mike Lange]
```

Comment (neokoenig) [reply CHANGELOG.md]: Since this didn't make it into the beta, can you start a new 2.1 section in the changelog?

Comment (chapmandu) [reply CHANGELOG.md]: Can do!

Comment (chapmandu) [reply CHANGELOG.md]: @neokoenig Is e1a4974 correct?

Comment (dbelanger) [reply CHANGELOG.md]: Hey @chapmandu, do we need to modify or add a test for this?

Comment (chapmandu) [reply CHANGELOG.md]: Yeah, I think we need a test to catch regressions.

Comment (chapmandu) [reply CHANGELOG.md]: Test case added

Comment (chapmandu) [reply CHANGELOG.md]: Looks like SQL Server doesn't like the CASE statement in the order by clause

File: wheels/tests/populate.cfm [hunk 3]
```diff
@@ -443,6 +454,10 @@ FROM users u INNER JOIN galleries g ON u.id = g.userid
 		,name="shop #local.i#"
 	)>
 </cfloop>
+<!--- shop 1 has 5 trucks! --->
```
Comment (chapmandu) [inline wheels/tests/populate.cfm]:
```suggestion
<!--- shop 1 has 5 trucks --->
```

**Comment by chapmandu** [comment]
If you chaps are happy with this I'll merge it into master

**Comment by neokoenig** [comment]
Go for it

**Pull request merged**